src/decide.ts

```diff
@@ -38,7 +38,9 @@ export class Decide {
             this.instance.requestRouter.endpointFor('api', '/decide/?v=3'),
             { data: encoded_data, verbose: true },
             { method: 'POST' },
-            (response) => this.parseDecideResponse(response as DecideResponse)
+            (response) => this.parseDecideResponse(response as DecideResponse),
```

Comment:
Ooooookay - time to refactor 😅 A function with a list of arguments like this should really turn into an object otherwise it is basically unreadable.

I'm gonna push some tidying to this branch 😉

Reply:
thank you 🙈🙈